resqpy/olio/relperm.py

```diff
@@ -0,0 +1,316 @@
+"""relperm.py: class for storing and retrieving dataframes of relative permeability
+data as RESQML objects.
+
+   note that this module uses the obj_Grid2dRepresentation class in a way that was not envisaged
+   when the RESQML standard was defined; software that does not use resqpy is unlikely to be
+   able to do much with data stored in this way
+"""
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+import os
+import logging
+import resqpy.olio.xml_et as rqet
+from resqpy.olio.dataframe import DataFrame
+
+version = '27th July 2021'
+
+log = logging.getLogger(__name__)
+log.debug(f'dataframe.py version {version}')
+
+
+class RelPerm(DataFrame):
+   """Class for storing and retrieving a pandas dataframe of relative permeability data.
+
+    note:
+       inherits from DataFrame class
+    """
+
+   def __init__(self,
+                model,
+                uuid = None,
+                df = None,
+                uom_list = None,
+                realization = None,
+                phase_combo = None,
+                low_sal = False,
+                table_index = None,
+                title = 'relperm_table',
+                column_lookup_uuid = None,
+                uom_lookup_uuid = None):
+      """Create a new RelPerm object from either a previously stored property or a pandas dataframe.
```

Comment:
'property' has a specific technical meaning in RESQML, probably better to use another word such as 'object' here.